dataland-frontend/src/components/resources/frameworkDataSearch/DataModelsTranslations.ts

```diff
@@ -0,0 +1,374 @@
+export const lksgKpis = {
```

Comment:
"amfori BSCI" und "Equal Opportunities And Non-Discrimination Policy" in Kategorie "Evidence, certificates and attestations" haben keinen Tooltip

Reply:
"Forced Labour And Slavery Prevention Training" in Kategorie "Forced labour, slavery and debt bondage" hat keinen Tooltip